core/deployment/src/main/resources/dev-ui/qwc-qosdk-controllers.js

```diff
@@ -0,0 +1,65 @@
+import { LitElement, html, css, nothing} from 'lit';
+import { JsonRpc } from 'jsonrpc';
+import '@vaadin/grid';
+import '@vaadin/details';
+import '@vaadin/list-box';
+import '@vaadin/item';
+import {columnBodyRenderer} from '@vaadin/grid/lit.js';
+export class ControllersWebComponent extends LitElement {
+
+    jsonRpc = new JsonRpc(this);
+
+    constructor() {
+        super();
+    }
+
+    render() {
+        return html`
+            <vaadin-grid .dataProvider="${this._controllersProvider}">
```

Comment:
Is there a reason you used a dataProvider ? This is usually usefull for very big sets of data. In most cases you can just set the data to a var and use the var. This allows you to re-set that var on hot reload and the screen will update.

Reply:
Changed the organization of the screen so this is now obsolete.